nginx.conf

```diff
@@ -91,7 +91,7 @@ http {
             proxy_http_version 1.1;
             proxy_set_header Host $http_host;
             proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection $connection_upgrade;
+            proxy_set_header Connection "upgrade";
```

Comment:
I'm not sure why this seems working. $connection_upgrade was defined in http block and seems to be correct.